### ISSUE TITLE
Is pyspark supported?

### ISSUE THREAD

**Issue by joaotnog** [microsoft/vscode-azurefunctions#3230]
When trying to initialize a Spark session with

`from pyspark.sql import SparkSession`
`aaa = SparkSession.builder.appName('SparkByExamples.com').getOrCreate()`

with a local HttpTrigger Azure Function deployed in VSCode, I get the error:

`Evaluating: aaa = SparkSession.builder.appName('SparkByExamples.com').getOrCreate() did not finish after 3.00 seconds.
This may mean a number of things:`
`- This evaluation is really slow and this is expected.
    In this case it's possible to silence this error by raising the timeout, setting the
    PYDEVD_WARN_EVALUATION_TIMEOUT environment variable to a bigger value.`
`- The evaluation may need other threads running while it's running:
    In this case, it's possible to set the PYDEVD_UNBLOCK_THREADS_TIMEOUT
    environment variable so that if after a given timeout an evaluation doesn't finish,
    other threads are unblocked or you can manually resume all threads.
    Alternatively, it's also possible to skip breaking on a particular thread by setting a
    `pydev_do_not_trace = True` attribute in the related threading.Thread instance
    (if some thread should always be running and no breakpoints are expected to be hit in it).`
`- The evaluation is deadlocked:
    In this case you may set the PYDEVD_THREAD_DUMP_ON_WARN_EVALUATION_TIMEOUT
    environment variable to true so that a thread dump is shown along with this message and
    optionally, set the PYDEVD_INTERRUPT_THREAD_TIMEOUT to some value so that the debugger
    tries to interrupt the evaluation (if possible) when this happens.`

Is pyspark supported in the Azure Functions extension or is there any config needed to fix it?

**Comment by alexweininger** [comment]
You may have better luck asking this question over on the main Azure Functions repository. https://github.com/Azure/Azure-Functions/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc

You can also try the Azure Functions stack overflow https://stackoverflow.com/questions/tagged/azure-functions

**Comment by nturinski** [comment]
Closing as out-of-scope for our extension.
